Entity/Comment.php

```diff
@@ -134,6 +141,37 @@ public function setThread(ThreadInterface $thread): CommentInterface
         return $this;
     }
 
+    public function getParent(): ?CommentInterface
+    {
+        return $this->parent;
+    }
+
+    public function setParent(?CommentInterface $parent = null): CommentInterface
+    {
+        $this->parent = $parent;
+
+        return $this;
+    }
+
+    public function getDepth(): int
+    {
+        return $this->depth;
+    }
+
+    public function getComments(): Collection
```

Comment:
is there a reason for calling this `getComments` instead of `getChildren`. would probably use `getChildren` here because we also have a `getParent` method.

Reply:
same as bellow :)